pdf/differential_test/main.py

```diff
@@ -6,208 +6,178 @@
 import io
```

Comment:
I'd like a feature output (non-HTML, that ends up in the FAW) when the artifact folders are all empty (e.g., nothing rendered the PDF).